app/js/actions.js

```diff
@@ -298,6 +298,13 @@ const toggleFullscreen = () => {
     })
 }
 
+const incrSearch = () => {
```

Comment:
Can you add this action to the unSupportedActions list in `app/js/input.js` so that it can't be called as an action by the user? Currently when I do `:call action.incrSearch` I start a search based on the current url. Also, please keep the name consistent with the setting: "incSearch" instead of "incrSearch", or give it a more readable name with full words such as "incrementSearch".

Reply:
Done and renamed to `incrementalSearch`.